packages/caliper-fabric/lib/connector-versions/v1/WalletFacade.js

```diff
@@ -0,0 +1,96 @@
+/*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+const IWalletFacade = require('../../identity-management/IWalletFacade');
+const ExportedIdentity = require('../../identity-management/ExportedIdentity');
+const {FileSystemWallet, InMemoryWallet, X509WalletMixin} = require('fabric-network');
+
+/**
+ * a Facade for the V1 Wallet implementation
+ */
+class WalletFacade extends IWalletFacade {
+
+    /**
+     */
+    constructor() {
+        super();
+        this.wallet = null;
+    }
+
+    /**
+     * initialize this WalletFacade
+     *
+     * @param {string} [walletPath] an optional path to a file system wallet
+     */
+    async initialize(walletPath) {
+        if (!walletPath) {
+            this.wallet = new InMemoryWallet();
+        } else {
+            this.wallet = new FileSystemWallet(walletPath);
+        }
+    }
+
+    /**
+     * Import an identity
+     *
+     * @param {string} mspId The mspId that owns the identity
+     * @param {string} identityName The name of the identity
+     * @param {string} certificate The identity certificate
+     * @param {string} privateKey The identity private key
+     */
+    async import(mspId, identityName, certificate, privateKey) {
+        const exists = await this.wallet.export(identityName);
```

Comment:
there is an exists call on the wallet that would be better just in case export throws an error if not found
```javascript
const exists = await this.wallet.exists(identityName)
```